CRM/Event/Page/ManageEvent.php

```diff
@@ -281,8 +281,8 @@ public function browse() {
       'String',
       $this
     );
-    $createdId = CRM_Utils_Request::retrieve('cid', 'Positive', $this, FALSE, 0);
```

Comment:
Just wondering if this is something to do with search not sure why your removing this

Reply:
It seems to be un-used in the browse() function?